src/ilcdlib/epd/factory.py

```diff
@@ -33,6 +34,7 @@ class EpdReaderFactory:
         "indata": IndataIlcdXmlEpdReader,
         "oekobau.dat": OekobauDatIlcdXmlEpdReader,
         "oekobaudat": OekobauDatIlcdXmlEpdReader,
+        "epditaly": EpdItalyIlcdXmlEpdReader,
```

Comment:
No need to make commits so granular. Just squash this with the one where you add dialect.